PocketKit/Tests/PocketKitTests/PocketAddTagsViewModelTests.swift

```diff
@@ -118,26 +120,85 @@ class PocketAddTagsViewModelTests: XCTestCase {
         XCTAssertNotNil(source.addTagsToSavedItemCall(at: 0))
     }
 
-    func test_recentTags_withTags_returnsRecentTags() {
+    func test_recentTags_withLessThanThreeTags_andPremiumUser_returnsNoRecentTags() {
```

Comment:
The test name doesn't match the implementation, which is a  little confusing. Within the test, there _are_ three tags, but the test name implies 2 or less, and the _true_ implementation of this functionality is > 3 tags.

Reply:
yes, you're right! thanks for the catch, updated `withThreeTags`. Wanted to capture if user had a total of three tags or less, then don't show recent tags.